src/admin/apiProvider/utils/listing.ts

```diff
@@ -103,3 +105,8 @@ export const apiListResponseToRAListResult = (response: ApiListResponse): Extend
     included: response?.included
   };
 };
+
+export const projectPitchesListResult = ({ data, indexTotal }: ProjectsPitchesConnection) => ({
+  data: data?.map((pitch: ProjectPitchDto) => ({ ...pitch, id: pitch.uuid })) as any,
+  total: indexTotal ?? 0
+});
```

Comment:
I'd argue that this  method isn't really needed, but even if you do want to keep it separate please move it to to the pitchDataProvider - it's not a general utility that will be needed in other places.